### PR TITLE
Toast notifications

### PR DESCRIPTION
**Pull request by lucasponce**
Capture missing errors from backend and show them into a toast pattern.

### CODE REVIEW

File: ui/src/main/ui/src/actions/actions-controller.js [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
-  function ($scope, $rootScope, $q, $modal, actionsService) {
+  .controller( 'hwk.actionsController', ['$scope', '$rootScope', '$q', '$modal', 'hwk.actionsService', 'Notifications',
+  function ($scope, $rootScope, $q, $modal, actionsService, Notifications) {
     'use strict';
 
     console.log("[Actions] Start: " + new Date());
     console.log("[Actions] $rootScope.selectedTenant " + $rootScope.selectedTenant);
 
+    var toastError = function (reason) {
+      console.log('[Actions] Backend error ' + new Date());
```
Comment (mtho11) [inline ui/src/main/ui/src/actions/actions-controller.js]: Do you still want the console.logs in the production code? Or maybe console.debug instead

Comment (lucasponce) [inline ui/src/main/ui/src/actions/actions-controller.js]: right, good point, console.debug is better are all these traces are just for debugging.

**Comment by lucasponce** [comment]
Thanks for your review @mtho11

**Comment by jshaughn** [comment]
Great! Nice job figuring out how to add error handling.

**Pull request merged**